tools/src/dbgen/Lib.hs

```diff
@@ -227,13 +229,23 @@ generateWalletDB CLI{..} spec@GenSpec{..} = do
 generateFakeTxs :: FakeTxsHistory -> AccountId -> UberMonad ()
 generateFakeTxs NoHistory _                = pure ()
 generateFakeTxs SimpleTxsHistory{..} aId   = do
-
-    db <- askWalletDB
-
     -- Get the number of txs we need to generate.
     let txsNumber = fromIntegral txsCount
 
-    accounts <- getAccounts Nothing -- aId
+    let batchSize = 100
+    let batches   = txsNumber `mod` batchSize
+    let remainder = txsNumber - (batches * batchSize)
+
+    void $ replicateM batches (generateNFakeTxs batchSize aId)
+    generateNFakeTxs remainder aId
```

Comment:
Could you explain this? `generateNFakeTxs` seems unhappy with generating too much entries?

Reply:
Just so we can persist the txs and everything related from memory to disk and clear the memory (GC).

Reply:
Aha, I see it. Then please specify in comment, why we don't generate all transactions at once, and why don't we generate and put 1 transaction at once.